test/functional/apps/management/_import_objects.ts

```diff
@@ -335,7 +334,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         // but as the initial popin can take a few ms before fading, we need to wait a little
         // to avoid clicking twice on the same modal.
         await delay(1000);
-        await PageObjects.common.clickConfirmOnModal(false);
+        await PageObjects.common.clickConfirmOnModal(true);
```

Comment:
This is what was causing the flakiness.

When adding the tests in FTR tests added in https://github.com/elastic/kibana/pull/76482, I had to add a manual delay between the first and second modal because the second one is appearing directly, so we can't rely on the `ensureHidden` option of `clickConfirmOnModal`. However I forgot to properly await the modal to close after the second click.